packages/cardpay-sdk/sdk/scheduled-payment-module.ts

```diff
@@ -424,4 +425,135 @@ export default class ScheduledPaymentModule {
 
     return requiredGas;
   }
+
+  async executeScheduledPayment(txnHash: string): Promise<SuccessfulTransactionReceipt>;
+  async executeScheduledPayment(
+    moduleAddressOrTxnHash: string,
+    tokenAddress: string,
+    amount: string,
+    payeeAddress: string,
+    fee: Fee,
+    executionGas: number,
+    maxGasPrice: string,
+    gasTokenAddress: string,
+    salt: string,
+    payAtOrRecurringDayOfMonth: number,
```

Comment:
Would it be better to split these into two optional params? Their values are two quite different things

Reply:
updated